sonar-iac-plugin/src/main/resources/static/documentation.md

```diff
@@ -1,22 +1,23 @@
 ---
-title: Infrastructure-as-Code
+title: Terraform/CloudFormation
 key: iac
 ---
 
 <!-- static -->
 <!-- update_center:iac -->
 <!-- /static -->
 
-
-
 ## Language-Specific Properties
 
-Discover and update the language-specific [properties](/analysis/analysis-parameters/)
+Discover and update the CloudFormation [properties](/analysis/analysis-parameters/) **<!-- sonarcloud -->Project <!-- /sonarcloud -->[Administration > General Settings > Cloudformation](/#sonarqube-admin#/admin/settings?category=cloudformation)**
 
-Cloudformation-specific properties in: <!-- sonarcloud -->Project <!-- /sonarcloud -->**[Administration > General Settings > Cloudformation](/#sonarqube-admin#/admin/settings?category=cloudformation)**
+Discover and update the Terraform [properties](/analysis/analysis-parameters/) **<!-- sonarcloud -->Project <!-- /sonarcloud -->[Administration > General Settings > Terraform](/#sonarqube-admin#/admin/settings?category=terraform)**
 
-Terraform-specific properties in: <!-- sonarcloud -->Project <!-- /sonarcloud -->**[Administration > General Settings > Terraform](/#sonarqube-admin#/admin/settings?category=terraform)**
+## Supported Versions, Formats and Providers
+* Terraform 1 (HCL/Terraform only)
```

Comment:
Shall we put "Terraform v1.0"?
Do we support Terraform v1.1?

Reply:
I would say we also support v1.2. As far as I can see, they haven't changed anything in the syntax. Our rules are mostly dependent on the providers like AWS, Azure, or GCP. They might change their behavior independently of Terraform versions.

Reply:
So we can probably say: Terraform v1.x (HCL format only)